matcher/service/openreview_interface.py

```diff
@@ -1,12 +1,42 @@
 import re
 import openreview
 import logging
+import redis
+import pickle
+import time
 from tqdm import tqdm
 
+class CacheHandler:
+    def __init__(self, host, port, db, cache_expiration, logger=logging.getLogger(__name__)):
+        self.redis_client = redis.Redis(host=host, port=port, db=db)
+        self.cache_expiration = cache_expiration
+        self.key_prefix = ''
+        self.logger = logger
+
+    def set_key_prefix(self, key_prefix):
+        self.key_prefix = key_prefix
+
+    def get_value(self, key):
+
+        redis_key = self.key_prefix + key
+        serialized_value = self.redis_client.get(redis_key)
+        if serialized_value:
+            self.logger.debug('CacheHandler get_value={}'.format(redis_key))
+            return pickle.loads(serialized_value)
+        return False
+
+    def set_value(self, key, value):
+        serialized_value = pickle.dumps(value)
+        self.redis_client.setex(self.key_prefix + key, self.cache_expiration, serialized_value)
+
 class ConfigNoteInterface:
-    def __init__(self, client, config_note_id, logger=logging.getLogger(__name__)):
+    def __init__(self, client, config_note_id, cache_handler, logger=logging.getLogger(__name__)):
         self.client = client
         self.logger = logger
+        self.cache = cache_handler
```

Comment:
I still do not see this class member self.cache being used anywhere. Can we remove it?